Week1/homework/js-exercises/10-compare-arrays.js

```diff
@@ -0,0 +1,19 @@
+// Exercise 10: Compare arrays
+
+'use strict';
+{
+
+let array1 = ['a','b','c','d'];
+let array2 = [1,2,3,4,5,6,7];
+
+//shows the length of each array
+console.log('The length of array1 is : ' + array1.length);
+console.log('The length of array2 is : ' + array2.length);
+
+//compare arrays
+if (array1.length == array2.length){
```

Comment:
Ah. You should strive for using triple equals(===) instead of double equals(==) when comparing things in JavaScript. This is because it is much safer to use triple equals because it checks not only the values but also the types of those variables.